action.yml

```diff
@@ -83,12 +83,15 @@ runs:
       shell: 'bash'
 
     - name: Version bump
+      uses: actions/github-script@v6
       id: version-bump
-      if: ${{ github.event_name == 'workflow_dispatch' }}
-      run: |
-        npm version --no-git-tag-version ${{ inputs.semver }}
-        echo "PACKAGE_VERSION=$(echo $(node -p "require('./package.json').version"))" >> $GITHUB_OUTPUT
-      shell: 'bash'
+      if: github.event_name == 'workflow_dispatch'
+      with:
+        result-encoding: string
+        github-token: ${{ inputs.github-token }}
+        script: |
+          const { getBumpedVersionNumber } = require('${{ github.action_path }}/dist/index.js')
```

Comment:
let's call this function `bumpVersion` which it more telling that it doesn't just get the version number, it also bumps it!